src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -400,6 +392,6 @@ def fake_quantize(
     """
     return _get_node_factory_opset13().create(
         "FakeQuantize",
-        as_nodes(data, input_low, input_high, output_low, output_high),
+        as_nodes(data, input_low, input_high, output_low, output_high, name=name),
```

Comment:
Does the name variable apply to all existing operations? The main idea is that the problem occurs not only for **FakeQuantizes**.

Reply:
@KodiaqQ this will be added to all ops separately, this PR is just a groundwork for it. Sometimes it will need adding `as_nodes`/`as_node` explicitly, so need to review all opsets starting from 1.